src/scripts/ExternalHelper.ts

```diff
@@ -100,17 +103,17 @@ class ExternalHelper {
         return containsDevRequirement;
     }
 
-    public static pokemonIsInLiveVersion(name: PokemonNameType) {
+    public static pokemonIsInDevelopment(name: PokemonNameType) {
         if (PokemonHelper.calcNativeRegion(name) > GameConstants.MAX_AVAILABLE_REGION) {
-            return false;
+            return true;
         }
 
         const locations = PokemonHelper.getPokemonLocations(name);
         if (!locations) {
```

Comment:
```suggestion
        if (!Object.keys(locations).length) {
```
`PokemonHelper.getPokemonLocations` always returns an object.
However, it will also return `true` Pokémon whose locations are not processed in the function yet (which should only happen to Alcremie, Milcery (Cheesy) and Exeggcute (Single), now).